docs/wiki/feature/atragmx.md

```diff
@@ -55,19 +61,23 @@ Horus ATragMX software considers atmospheric conditions, gun data, ammunition, r
 - Optionally, `Save Gun` and `Done` in the `GunList`.
  
 **In position:**
+
 - Update the `Atmsphr` column with the [Kestrel 4500]({{ site.baseurl }}/wiki/feature/kestrel4500.html) and `Done`.
 - *Check the new `Muzzle Velocity` in the `Gun` column.*
 - Update the `Target` column (the [wind arrow]({{ site.baseurl }}/wiki/feature/weather.html) will also help). [[1]](https://horusvision.com/download/manual_Horus_ATrag-v385.pdf#page=16) [[2]](https://horusvision.com/download/manual_Horus_ATrag-v385.pdf#page=30) [[3]](https://horusvision.com/download/manual_Horus_ATrag-v385.pdf#page=32) [[4]](https://horusvision.com/download/manual_Horus_ATrag-v385.pdf#page=33)
+- *The latitude for all common maps can be found in the [ACE3 Github](https://github.com/acemod/ACE3/blob/master/addons/common/functions/fnc_getMapData.sqf).*
```

Comment:
Like that, correct ?
`{{ site.ace.githubUrl }}/blob/master/addons/common/functions/fnc_getMapData.sqf`